src/controller_examples/fluent_controller/fluentbit/exec/resource/daemon_set.rs

```diff
@@ -226,7 +236,7 @@ fn make_fluentbit_pod_spec(fb: &FluentBit) -> (pod_spec: PodSpec)
                 });
                 proof {
                     assert_seqs_equal!(
-                        volume_mounts@.map_values(|volume_mount: VolumeMount| volume_mount@),
+                        volume_mounts@.map_values(|volume_mount: VolumeMount| volume_mount@) ==
```

Comment:
What is this change for?

Reply:
Sorry. I changed `assert_seqs_equal!` to `assume` before, so I also made this change.

Reply:
And I just realized that assert_seqs_equal supports both styles. Could you revert this back for now?